benchmark/package.json

```diff
@@ -28,14 +26,6 @@
   },
   "author": "",
   "license": "MIT",
-  "files": [
```

Comment:
I am afraid the new npmignore is not an equivalent replacement for this `files` section.